### PR TITLE
Bring Ep07 to Ep10 to new template

### PR DESCRIPTION
**Pull request by rogerkuou**
This PR focus on bringing the most updated ([9b03f489](https://github.com/esciencecenter-digital-skills/geospatial-python/commit/9b03f489b0cb854a8f2e744f04414e271d851e14)) content of Ep07-10 to new template format.

- [x] Restore the original index number to include the introduction eps
- [x] Ep 07-vector-data-in-python
- [x] Ep 08-crop-raster-data
- [x] Ep 09-raster-calculations
- [x] Ep 10-zonal-statistics

### CODE REVIEW

File: episodes/09-raster-calculations.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ where $NIR$ and $red$ label the reflectance values of the corresponding waveleng
 Values close to one indicate high density of green leaves. Poorly vegetated areas typically have NDVI values close to
 zero. Negative NDVI values often indicate cloud and water bodies.
 
-![PONE-NDVI Source: Wu C-D, McNeely E, Cedeño-Laurent JG, Pan W-C, Adamkiewicz G, Dominici F, et al. (2014) Linking Student Performance in Massachusetts Elementary Schools with the “Greenness” of School Surroundings Using Remote Sensing. PLoS ONE 9(10): e108548. https://doi.org/10.1371/journal.pone.0108548](../fig/E09-01-PONE-NDVI.jpg)
+![PONE-NDVI. Source: Wu C-D, McNeely E, Cedeño-Laurent JG, Pan W-C, Adamkiewicz G, Dominici F, et al. (2014) Linking Student Performance in Massachusetts Elementary Schools with the “Greenness” of School Surroundings Using Remote Sensing. PLoS ONE 9(10): e108548. https://doi.org/10.1371/journal.pone.0108548](fig/E09/PONE-NDVI.jpg){alt="PONE-NDVI image"}
```
Comment (fnattino) [inline episodes/09-raster-calculations.md:37]:
```suggestion
![Source: Wu C-D, McNeely E, Cedeño-Laurent JG, Pan W-C, Adamkiewicz G, Dominici F, et al. (2014) Linking Student Performance in Massachusetts Elementary Schools with the “Greenness” of School Surroundings Using Remote Sensing. PLoS ONE 9(10): e108548. https://doi.org/10.1371/journal.pone.0108548](fig/E09/PONE-NDVI.jpg){alt="PONE-NDVI image"}
```

File: episodes/09-raster-calculations.md [hunk 2]
```diff
@@ -254,7 +254,7 @@ First, we define NDVI classes based on a list of values, as defined in the last
 `[-1, 0., 0.2, 0.7, 1]`. When bins are ordered from
 low to high, as here, `numpy.digitize` assigns classes like so:
 
-![NDVI-classes Source: Image created for this lesson ([license]({{ page.root }}{% link LICENSE.md %}))](../fig/E09-08-NDVI-classes.jpg)
+![NDVI-classes. Source: Image created for this lesson ([license]({{ page.root }}{% link LICENSE.md %}))](fig/E09/NDVI-classes.jpg){alt="NDVI classes"}
```
Comment (fnattino) [inline episodes/09-raster-calculations.md:258]:
```suggestion
![Source: Image created for this lesson ([license](../LICENSE.md))](fig/E09/NDVI-classes.jpg){alt="NDVI classes"}
```

File: episodes/07-vector-data-in-python.md [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+- Load spatial objects.
+- Select the spatial objects within a bounding box.
+- Perform a CRS conversion of spatial objects.
+- Select features of spatial objects.
+- Match objects in two datasets based on their spatial relationships.
+:::
+
+## Introduction
+
+As discussed in [Episode 2: Introduction to Vector Data](../02-intro-vector-data.md/), vector data represents specific features on the Earth's surface using points, lines, and polygons. These geographic elements can then have one or more attributes assigned to them, such as 'name' and 'population' for a city, or crop type for a field. Vector data can be much smaller in (file) size than raster data, while being very rich in terms of the information captured.
```
Comment (fnattino) [inline episodes/07-vector-data-in-python.md:22]:
```suggestion
As discussed in [Episode 2: Introduction to Vector Data](../02-intro-vector-data.md), vector data represents specific features on the Earth's surface using points, lines, and polygons. These geographic elements can then have one or more attributes assigned to them, such as 'name' and 'population' for a city, or crop type for a field. Vector data can be much smaller in (file) size than raster data, while being very rich in terms of the information captured.
```

**Review by fnattino — CHANGES_REQUESTED** [review]
Great @rogerkuou ! I spotted a couple of broken links and I have committed a change in order to add a space below a figure (could not comment on it, because was outside the region you touched).

**Comment by rogerkuou** [comment]
Thanks @fnattino ! I applied your comments and will merge

**Pull request merged**